spec/models/category_spec.rb

```diff
@@ -0,0 +1,5 @@
+require "spec_helper"
+
+describe Category do
+  it { should have_many(:videos) }
```

Comment:
Instead of `should` you could even use `is_expected.to`. The latter is actually new way RSpec decided to go.